src/router.ts

```diff
@@ -174,12 +181,17 @@ export function createRouterMock(options: RouterMockOptions = {}): RouterMock {
     return pendingNavigation || Promise.resolve()
   }
 
+  function setParams(params: RouteParams) {
+    router.currentRoute.value = { ...router.currentRoute.value, params }
```

Comment:
shouldn't `router.currentRoute.value = router.resolve({ params })` work too? It will also allow using `RouteParamsRaw` as `params` and allow numbers while still casting them to strings

Reply:
I can definitely use `router.resolve({ params })` 👍

But can I use `RouteParamsRaw`? It looks like it's not publicly exported

`vue-router.d.ts`:
```ts
export declare type RouteParams = Record<string, RouteParamValue | RouteParamValue[]>;
declare type RouteParamsRaw = Record<string, RouteParamValueRaw | RouteParamValueRaw[]>;
```

Reply:
I will export it